### PR TITLE
[CHORE] feature request && bug issue templates

### PR DESCRIPTION
**Pull request by emekdahl**
This PR provides templates for feature request and bug issue templates in response to @chamini2 's comment about liking the format of this recent [bug report](https://github.com/fal-ai/fal/issues/282).

### CODE REVIEW

**Review by chamini2 — APPROVED** [review]
This is super cool! Thank you. I will wait for some others of the team to check out the wording before merging.

File: .github/workflows/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+---
+
+**Describe the bug**
+Please describe the issue.
+
+**Your environment**
+- `fal` version: 
+- OS:
+- Other relevant dependencies & versions:
+
```
Comment (chamini2) [inline .github/workflows/ISSUE_TEMPLATE/bug_report.md:17]: Could we change this to:

~~~suggestion
**Your environment**
- OS: 
- Paste the following commands output:
  ```
  fal --version
  dbt --version
  ```
- Adapter being used: <!-- (if more than one dbt plugin installed) --> 

~~~

To get more information needed to reproduce.

Comment (emekdahl) [reply .github/workflows/ISSUE_TEMPLATE/bug_report.md]: That's a good idea, thanks!

**Review by turbo1912 — APPROVED** [review]
Thanks @emekdahl !

**Review by mederka — APPROVED** [review]
This is awesome!

**Comment by chamini2** [comment]
Thank you @emekdahl !

**Pull request merged**